aws/kms/client.go

```diff
@@ -0,0 +1,19 @@
+package kms
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/kms"
+)
+
+func NewKMSClient(ctx context.Context, region string) (*kms.Client, error) {
+	config, err := config.LoadDefaultConfig(ctx, config.WithRegion(region))
```

Comment:
Guessing LoadDefaultConfig will read aws credentials from env vars? Why not use GetAWSConfig from aws/config.go instead?

Reply:
As commented above, I think its simpler to use the method aws sdk provides

Reply:
using LoadDefaultConfig is recommended. ref: https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/